modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/StandardHeader.scala

```diff
@@ -0,0 +1,90 @@
+package edu.gemini.seqexec.server
+
+import edu.gemini.seqexec.model.dhs.ObsId
+
+import edu.gemini.seqexec.server.DhsClient._
+
+/**
+  * Created by jluhrs on 1/31/17.
+  */
+
+trait TcsKeywordsReader {
```

Comment:
Did you consider to have a single `get[A](name: String):A` type of method instead?